Assets/ConsentManagementProvider/Scripts/interface/ISpSdk.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+namespace ConsentManagementProviderLib
+{
+    public interface ISpSdk
+    {
+        void Initialize(
+            int accountId, 
+            int propertyId, 
+            string propertyName,
+            MESSAGE_LANGUAGE language, 
+            string gdprPmId, 
+            string ccpaPmId,
+            string usnatPmId,
+            List<SpCampaign> spCampaigns,
+            CAMPAIGN_ENV campaignsEnvironment, 
+            long messageTimeoutInSeconds = 3,
+            bool? transitionCCPAAuth = null,
+            bool? supportLegacyUSPString = null);
```

Comment:
We should move 
```
bool? transitionCCPAAuth = null,
bool? supportLegacyUSPString = null
```
to `SpCampaign` class.

Reply:
Done in https://github.com/SourcePointUSA/unity-sdk/pull/61